tests/test_connected_account.py

```diff
@@ -119,7 +121,10 @@ def test_update_connected_account(account_retrieve, account_update, managed_acco
 
     assert response.status_code == 200, response.data
     assert response.data["owner"] == managed_account.owner.id
-    assert response.data["source"]["business_name"] == data["business_name"]
+    if isinstance(response.data["source"], str):
```

Comment:
I'm not 100% sure about these changes, which are effective with Stripe 2.
It doesnt seem important enough to create backwards compatibility.
Happy to be guided here.

Reply:
Ah, I wonder if this is due to the change to `jsonfield`

Reply:
I have removed this hack, so this test currently fails.
Maybe need to recreate a testbed equal to ~2 years ago to check what happened when the commit was authored.